src/components/host/host.css

```diff
@@ -42,9 +42,10 @@ top-section{
 }
 
 .person{
```

Comment:
Should we call them person or human? Maybe when we do a clean up we should probably decide and make it more consistent?